crowbar_framework/config/routes.rb

```diff
@@ -240,4 +236,48 @@
   # TODO(must): Get rid of this wildcard route
   match "/:controller/:action/*(:.format)",
     via: [:get, :post, :put, :patch, :delete]
+
+  namespace :api,
```

Comment:
@aspiers The idea behind the split is to completely get rid of the `ui/backups_controller` once we switch to the Angular UI, and from then on only use the `api/backups_controller`. If we wouldn't use the Rails views anymore we could simply drop it